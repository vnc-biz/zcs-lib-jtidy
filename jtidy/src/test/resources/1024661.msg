<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test 1024661-->
<messages>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[
Tidy (vers 4th August 2004) Parsing "InputStream"
]]></text>
  </message>
  <message>
    <code>68</code>
    <level>2</level>
    <line>7</line>
    <column>1</column>
    <text><![CDATA[<table> joining values of repeated attribute "style"]]></text>
  </message>
  <message>
    <code>110</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Doctype given is "-//W3C//DTD XHTML 1.0 Strict//EN"]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like XHTML 1.0 Strict]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[1 warning, no errors were found!
]]></text>
  </message>
</messages>
