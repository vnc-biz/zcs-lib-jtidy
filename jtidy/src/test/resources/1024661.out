<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[1024661] Error Parsing duplicate style</title>
</head>
<body>
<table summary="none" style="font-size: 1pt; font-size: 1pt">
<tr>
<td>...</td>
</tr>
</table>
</body>
</html>
